### PR TITLE
Feature/ted 605

### PR DESCRIPTION
**Pull request by idellal**

### CODE REVIEW

File: mappings/package_F03/transformation/mappings/annex_d1.rml.ttl [hunk 1]
```diff
@@ -28,8 +28,45 @@ tedm:Procedure a rr:TriplesMap ;
...
+        [
+            rr:predicate epo:hasDirectAwardJustification ;
+            rr:objectMap
+                [
+                    rml:reference "if (exists(PT_AWARD_CONTRACT_WITHOUT_CALL/D_ACCORDANCE_ARTICLE/D_PROC_OPEN) or exists (PT_AWARD_CONTRACT_WITHOUT_CALL/D_ACCORDANCE_ARTICLE/D_PROC_RESTRICTED)) then 'http://publications.europa.eu/resource/authority/direct-award-justification/unsuitable' else null";
+                    rr:termType rr:IRI;
+                ]
+
+        ] ;
+    rr:predicateObjectMap
```
Comment (csnyulas) [inline mappings/package_F03/transformation/mappings/annex_d1.rml.ttl:60]: This not need to be here. It is duplicated by mistake.

File: src/mappings/annex_d1.rml.ttl [hunk 2]
```diff
@@ -28,8 +28,45 @@ tedm:Procedure a rr:TriplesMap ;
...
+        [
+            rr:predicate epo:hasDirectAwardJustification ;
+            rr:objectMap
+                [
+                    rml:reference "if (exists(PT_AWARD_CONTRACT_WITHOUT_CALL/D_ACCORDANCE_ARTICLE/D_PROC_OPEN) or exists (PT_AWARD_CONTRACT_WITHOUT_CALL/D_ACCORDANCE_ARTICLE/D_PROC_RESTRICTED)) then 'http://publications.europa.eu/resource/authority/direct-award-justification/unsuitable' else null";
+                    rr:termType rr:IRI;
+                ]
+
+        ] ;
+    rr:predicateObjectMap
```
Comment (csnyulas) [inline src/mappings/annex_d1.rml.ttl:60]: This not need to be here. It is duplicated by mistake.

**Review by csnyulas — APPROVED** [review]
I approve this, as it addresses very well the main issue. There are a couple minor issues that I will fix in a separate commit.

**Review by costezki — APPROVED** [review]

File: mappings/package_F03/transformation/mappings/annex_d1.rml.ttl (hunk 1 above)
Comment (csnyulas) [reply mappings/package_F03/transformation/mappings/annex_d1.rml.ttl:60]: Was fixed in commit https://github.com/meaningfy-ws/ted-sws-artefacts/pull/88/commits/23bd245065f233aa5edd2f16dc4fe3b12cfb57ca .

File: src/mappings/annex_d1.rml.ttl (hunk 2 above)
Comment (csnyulas) [reply src/mappings/annex_d1.rml.ttl:60]: Was fixed in commit https://github.com/meaningfy-ws/ted-sws-artefacts/pull/88/commits/23bd245065f233aa5edd2f16dc4fe3b12cfb57ca .

**Pull request merged**